lib/doorkeeper/models/concerns/revocable.rb

```diff
@@ -8,6 +8,30 @@ def revoke(clock = Time)
       def revoked?
         !!(revoked_at && revoked_at <= Time.now)
       end
+
+      def revoke_in(time)
+        update_attribute :revoked_at, Time.now + time
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.